readthedocs/doc_builder/python_environments.py

```diff
@@ -355,6 +355,7 @@ def setup_base(self):
             'conda',
             'env',
             'create',
+            '--quiet',
```

Comment:
I  think we don't want `--quiet` here. It made sense the first time I read the docs, but @jorisvandenbossche made me realized that it does not show all the packages installed either (only when called with `conda env`). More discussion here https://github.com/rtfd/readthedocs.org/issues/5099#issuecomment-453922323

Reply:
Note that without `--quiet`, it *also* does not show all the packages installed. It only shows the download progress output. In some way you can derive from that the installed packages, but only for short package names (otherwise the name and certainly the version is truncated), and it is also in a very unreadable way (compared to `conda list` or the way `conda create` lists the packages to be installed). 
So given that, I would personally still be in favor of using `--quiet` here, as it simply removes somewhat non-useful output.

Reply:
See eg the output of the `conda env create` step in https://readthedocs.org/projects/geopandas/builds/8384815/: for quite some of the packages in the download progress, you can't see the full version, and for some not even the full package name.